app/views/layouts/application.html.erb

```diff
@@ -47,37 +47,11 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
 </head>
 <body class='<%= controller_class %>'>
-  <header class="usa-site-header" role="banner">
-
-    <div class="usa-disclaimer">
-      <div class="usa-grid">
-        <div class="usa-width-one-whole">
-          <span class="usa-disclaimer-official">
-            <%= image_tag 'us_flag_small.png', class: 'usa-flag_icon', alt: "US flag signifying that this is a United States Federal Government website" %>
-            An official website of the United States Government
-          </span>
-        </div>
-      </div>
-    </div>
 
-    <nav class="usa-site-navbar">
-      <div class="usa-grid header-table">
-        <div class="logo header-row" id="logo">
-          <div class="header-image">
-            <%= image_tag "18F-Logo-2016-Blue.svg", alt: "18f-logo" %>
-          </div>
-          <div class="header-title-text">
-            <a href="/" accesskey="1" title="Home" aria-label="Home">Micropurchase</a>
-          </div>
-          <button id="button-nav-drawer" class="usa-button usa-button-outline-inverse" data-drawer-toggle="menu"><span class="fa fa-bars"></span></button>
-          <div class="header-account">
-            <%= render partial: current_user.decorate.nav_drawer_partial,
-              locals: { current_user: current_user } %>
-          </div>
-        </div>
-      </div>
-    </nav>
-  </header>
+  <%= render partial: 'components/nav_drawer', locals: { current_user: current_user } %>
```

Comment:
@jessieay it is rendering, it just isn't viewable unless you are in mobile and click on the menu bars icon